library/accessory_workflows/build_bwa_reference/bwa-mk-index.wdl

```diff
@@ -0,0 +1,52 @@
+version 1.0
+
+workflow BuildBWARef {
+  input {
+      String ref_name
+      File reference_fasta
+      File chrom_sizes_file
+  }
+
+  call BuildBWAreference {
+     input:
+	    ref_name = ref_name,
+            reference_fasta = reference_fasta,
+            chrom_sizes_file = chrom_sizes_file
+	}
+
+	output {
+	    File referenceBundle = BuildBWAreference.referenceBundle
+	}
+}
+
+task BuildBWAreference {
+     input {
+        String ref_name ## name of the tar.bz2 files without the suffix
+        File reference_fasta
+        File chrom_sizes_file
+     }
+
+     command <<<
+        mkdir genome
+        mv ~{chrom_sizes_file} genome/chrom.sizes
+        if [ ${file: -3} == ".gz" ]
```

Comment:
suggest using the `file` utility to do this. See: https://github.com/HumanCellAtlas/skylab/pull/224/files#diff-bef35d1a455bc541c5db7559c3289e07R83